Explorer/Assets/DCL/Character/CharacterCamera/Systems/ApplyCinemachineSettingsSystem.cs

```diff
@@ -0,0 +1,68 @@
+using Arch.Core;
+using Arch.System;
+using Arch.SystemGroups;
+using Cinemachine;
+using DCL.CharacterCamera;
+using DCL.CharacterCamera.Components;
+using DCL.DebugUtilities;
+using DCL.DebugUtilities.UIBindings;
+using ECS.Abstract;
+using System;
+using UnityEngine;
+using UnityEngine.UIElements;
+
+namespace DCL.Character.CharacterCamera.Systems
+{
+    [UpdateInGroup(typeof(CameraGroup))]
+    public partial class ApplyCinemachineSettingsSystem : BaseUnityLoopSystem
```

Comment:
Does this need to UpdateBefore in the camera group?

Reply:
I thought about this but In the end I had no issues of this being executed before or after other systems